<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BCipherClass</class>
 <widget class="QMainWindow" name="BCipherClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BCipher</string>
  </property>
  <property name="windowIcon">
   <iconset resource="BCipher.qrc">
    <normaloff>:/Resources/lock.svg</normaloff>:/Resources/lock.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="encryptionBtn">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="BCipher.qrc">
          <normaloff>:/Resources/lock.svg</normaloff>:/Resources/lock.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="encryptionLbl">
        <property name="text">
         <string>Mã hóa</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="decryptionBtn">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="BCipher.qrc">
          <normaloff>:/Resources/unlock.svg</normaloff>:/Resources/unlock.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="decryptionLbl">
        <property name="text">
         <string>Giải mã</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="informationBtn">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="BCipher.qrc">
          <normaloff>:/Resources/information.svg</normaloff>:/Resources/information.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="informationLbl">
        <property name="text">
         <string>Thông tin</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>394</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="EncryptionWidget" name="encryptionPage"/>
      <widget class="DecryptionWidget" name="decryptionPage"/>
      <widget class="QWidget" name="informationPage">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPlainTextEdit" name="plainTextEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="plainText">
           <string>BCipher 1.0.1, 2019
Công cụ mã hóa các file dữ liệu vừa và nhỏ. 
Thuật toán sử dụng AES có chiều dài khóa 256 và phương thức kiểu CFB. 
--
(C) Nguyễn Sỹ Bách, bachns.dev@gmail.com</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>EncryptionWidget</class>
   <extends>QWidget</extends>
   <header location="global">EncryptionWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DecryptionWidget</class>
   <extends>QWidget</extends>
   <header location="global">DecryptionWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="BCipher.qrc"/>
 </resources>
 <connections/>
</ui>
